src/components/pages/HomeView.tsx

```diff
@@ -0,0 +1,184 @@
+import React, { Component } from 'react';
+import globalize from '../../scripts/globalize';
+import LibraryMenu from '../../scripts/libraryMenu';
+import { clearBackdrop } from '../backdrop/backdrop';
+import layoutManager from '../layoutManager';
+import * as mainTabsManager from '../maintabsmanager';
+import '../../elements/emby-tabs/emby-tabs';
+import '../../elements/emby-button/emby-button';
+import '../../elements/emby-scroller/emby-scroller';
+
+type IProps = {
+    tab?: string;
+}
+
+type OptionsProps = {
+    autoFocus?: boolean;
+    refresh?: boolean
+}
+
+type ControllerProps = {
+    onResume: (
+        options: OptionsProps
+    ) => void;
+    refreshed: boolean;
+    onPause: () => void;
+    destroy: () => void;
+}
+
+class HomeView extends Component <IProps> {
```

Comment:
Is there any reason to use a class component instead of a function component for this?

Reply:
No just implemented the way before.

Reply:
@thornbill update to function components https://github.com/jellyfin/jellyfin-web/pull/3567/commits/c1ea22268bc956d34e491059ef10da1ead28d68c